runtime/js/telemetry.js

```diff
@@ -192,13 +223,28 @@ class Span {
     this.enter();
   }
 
+  get traceId() {
```

Comment:
these properties don't exist in otel. if buffers are fastest we could just do `this.traceId = buffer` without using a private. if/when we do perf work in upstream otel, we could do similar, with strings only being created in places where the api expects it.

Reply:
We could also remove these fields entirely then and only have the `spanContext()` API?

Reply:
well, the original idea was to avoid needing to use `spanContext`, since the extra object allocation is theoretically unneeded when flushing a span.